scripts/metrics/eval.py

```diff
@@ -4,17 +4,38 @@
 
 from moses.metrics.metrics import get_all_metrics
 from moses.script_utils import read_smiles_csv
+import numpy as np
```

Comment:
it's better to write first standard than third party and finally local imports
https://www.python.org/dev/peps/pep-0008/#imports